go/client/cmd_simplefs_check.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSCheck is the 'simplefs srar' command.
+type CmdSimpleFSCheck struct {
+	libkb.Contextified
+	opid keybase1.OpID
+}
+
+// NewCmdSimpleFSCheck creates a new cli.Command.
+func NewCmdSimpleFSCheck(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "check",
+		ArgumentHelp: "<opid>",
+		Usage:        "check pending operation",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdDeviceList{Contextified: libkb.NewContextified(g)}, "check", c)
+		},
+	}
+}
+
+// RunClient runs the command in client/server mode.
+func (c *CmdSimpleFSCheck) Run() error {
+	cli, err := GetSimpleFSClient(c.G())
+	if err != nil {
+		return err
+	}
+
+	progress, err := cli.SimpleFSCheck(context.TODO(), c.opid)
+	if err != nil {
+		return err
+	}
+
+	w := GlobUI.DefaultTabWriter()
```

Comment:
It should be possible not to use this. I think there's a `UI` dangling off of `G`.